security/bouncycastle-fips/src/main/java/io/quarkus/ts/security/bouncycastle/fips/BouncyCastleFipsEndpoint.java

```diff
@@ -0,0 +1,27 @@
+package io.quarkus.ts.security.bouncycastle.fips;
+
+import java.security.Security;
+import java.security.Signature;
+import java.util.Arrays;
+import java.util.stream.Collectors;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+
+@Path("/api")
+public class BouncyCastleFipsEndpoint {
+    @GET
+    @Path("listProviders")
```

Comment:
Can we add slashes to the names of subpaths? So it would be something like "/listProviders"?